codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/RubyCodeWriter.java

```diff
@@ -377,7 +408,7 @@ public RubyCodeWriter apply(String filename, String namespace) {
     }
 
     /**
-     * Adds TypeScript symbols for the "$T" formatter.
+     * Adds Ruby symbols for the "$T" formatter.
```

Comment:
Haha, whoops.  I wonder where this code came from....

Reply:
Haha I wonder...